src/testdir/test_filetype.vim

```diff
@@ -2874,4 +2873,16 @@ func Test_org_file()
   filetype off
 endfunc
 
+" Filetypes detected from names of existing files
+func Test_pacmanlog()
+  filetype on
+  for fname in ['pacman.log', 'pacman.log.1', 'pacman.log-20250123']
+    call writefile(["[2025-01-23T01:23:45+0000] [PACMAN] Running 'pacman -S -y --config /etc/pacman.conf --'"], fname, 'D')
+    exe 'split ' .. fname
+    call assert_equal('pacmanlog', &filetype, 'for text: ' .. string(fname))
+    bwipe!
+  endfor
+  filetype off
+endfunc
```

Comment:
I believe that a user opens a log file but does not newly create it, therefore I remove `BufNewFile` event, which led to test failure.
File content is not essential and can be empty.

Or should I restore `BufNewFile`?

Reply:
So basically you are saying, that those pacman.log.* are created by some automatic processes but not by a human? that is probably fine, but I think it would be better to change it like this then:
```
au BufNewFile,BufRead pacman.log			setf pacmanlog
" pacman.log.1 are not supposed to be created by users, but should be detected nevertheless
au BufRead pacman.log*					call s:StarSetf('pacmanlog')
```

Reply:
Yes, pacman.log is literally a log file for pacman, and a user will not create logrotate’d ones.